spec/lib/rbac/filterer_spec.rb

```diff
@@ -74,6 +74,12 @@
         expect(results).to match_array [owned_vm]
       end
 
+      it "with :extra_cols finds Service" do
+        FactoryGirl.create :service, :evm_owner => owner_user
+        results = described_class.search(:class => "Service", :extra_cols => [:owned_by_current_user]).first
+        expect(results.first.attributes["owned_by_current_user"]).to be false
```

Comment:
Yeah, that `virtual_attribute` is a bit weird in the sense that it makes it a `boolean` result based on existence.  There is a bit lengthy-er of an explanation there of some edge cases on how that result can be changed based on `JOIN` clauses (`LEFT JOIN` versus `JOIN`), but point being is it is there to check that the current user is the owner, and have it return a result column of a boolean (`true` being this VM is owned by the current user, and this is mostly used as a filter, not a column, in a report).